dev-java/asm/asm-3.3.2.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+JAVA_PKG_IUSE="source test"
+MAVEN_ID="org.ow2.asm:asm:3.3.2"
+JAVA_TESTING_FRAMEWORKS="junit-4"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Bytecode manipulation framework for Java"
+HOMEPAGE="https://asm.ow2.io"
+MY_P="ASM_${PV//./_}"
+SRC_URI="https://gitlab.ow2.org/asm/asm/-/archive/${MY_P}/asm-${MY_P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="3"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86 ~x64-macos"
+
+# grep 'does not exist' /var/tmp/portage/dev-java/asm-3.3.2/temp/build.log | cut -d':' -f4 | sort | uniq
+# package alt.jiapi.reflect does not exist
+# package com.claritysys.jvm.builder does not exist
+# package com.claritysys.jvm.classfile does not exist
+# package gnu.bytecode does not exist
+# package gnu.bytecode.Type does not exist
+# package jbet does not exist
+# package org.mozilla.classfile does not exist
+# package serp.bytecode does not exist
+# package TypeDesc does not exist
+#	RESTRICT="test"
+
+DEPEND=">=virtual/jdk-1.8
+	test? (
+		dev-java/aspectj:0
+		dev-java/bcel:0
+		dev-java/cojen:0
+		dev-java/janino:0
+		dev-java/javassist:3
+		dev-java/jclasslib:0
+		dev-java/junit:0
```

Comment:
you have `junit:0` here but `junit-4` in `JAVA_TESTING_FRAMEWORKS`. i suppose the slots should be the same.

Reply:
> you have `junit:0` here but `junit-4` in `JAVA_TESTING_FRAMEWORKS`. i suppose the slots should be the same.

Adjusted. Still missing some dependencies:
```
 package alt.jiapi.reflect does not exist
 package com.claritysys.jvm.builder does not exist
 package com.claritysys.jvm.classfile does not exist
 package gnu.bytecode does not exist
 package gnu.bytecode.Type does not exist
 package jbet does not exist
 package org.mozilla.classfile does not exist
 package serp.bytecode does not exist
```
Any idea where to find `gnu.bytecode`or `com.claritysys.jvm`?